src/main/java/org/jabref/gui/filelist/LinkedFilesEditDialogViewModel.java

```diff
@@ -61,17 +61,17 @@ private void checkExtension() {
     }
 
     public void openBrowseDialog() {
-        String fileText = linkProperty().get();
+        String fileText = link().get();
 
         Optional<Path> file = FileHelper.expandFilename(database, fileText, preferences.getFileDirectoryPreferences());
 
         Path workingDir = file.orElse(Paths.get(preferences.get(JabRefPreferences.WORKING_DIRECTORY)));
         String fileName = Paths.get(fileText).getFileName().toString();
 
         FileDialogConfiguration fileDialogConfiguration = new FileDialogConfiguration.Builder()
-        .withInitialDirectory(workingDir)
-        .withInitialFileName(fileName)
-        .build();
+                                                                                               .withInitialDirectory(workingDir)
```

Comment:
@tobiasdiez  this is how the indentation at column level works: As I said before it's not possible to align it at the dot position. So when I use indencation on column  all things are then wrapped after Builder (Github doesn't show it that wide).